core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -130,13 +129,13 @@ trait RawlsBillingProjectComponent {
     def loadAllUsersWithProjects: ReadAction[Map[RawlsUser, Iterable[RawlsBillingProjectName]]] = {
```

Comment:
Maybe change the name of this function to "loadAllUsersAndTheirProjects" so that it's clear that it's not loading just users that have projects